.github/workflows/push.yml

```diff
@@ -496,6 +496,15 @@ jobs:
           # Snowflake
           DRIVERS_TESTS_SNOWFLAKE_CUBEJS_DB_USER: ${{ secrets.DRIVERS_TESTS_CUBEJS_DB_SNOWFLAKE_USER }}
           DRIVERS_TESTS_SNOWFLAKE_CUBEJS_DB_PASS: ${{ secrets.DRIVERS_TESTS_CUBEJS_DB_SNOWFLAKE_PASS }}
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v5
+        if: (matrix.node-version == '22.x')
```

Comment:
We need to add condition for `transpile-worker-threads`, because otherwise we will upload it two times. right?

Reply:
@KSDaemon but it's defined in unit job.

![image](https://github.com/user-attachments/assets/a91e7303-5428-4d99-b84f-2ca859bdc3b5)

Reply:
But you posted a comment not to this place but for `integration` job :))))

Reply:
fixed

Reply:
Oh, lol, It's two different places :D Anyway, fixed :D